bsi/src/main/java/org/roaringbitmap/bsi/BitmapSliceIndex.java

```diff
@@ -75,8 +75,14 @@ enum Operation {
    * currentMaxValue/currentMinValue are optional,it's can be compute from input value list.
    * and avoiding bsi expend slice array capacity.
    */
+  @Deprecated
   void setValues(List<Pair<Integer, Integer>> values, Integer currentMaxValue, Integer currentMinValue);
```

Comment:
Deprecated this as I think it's more natural to calculate the min-max from the actual values, rather than setting them externally, such operations should be handed over to a capacity expansion interface.